docs/controllers/services/annotations.md

```diff
@@ -180,4 +180,20 @@ You have to supply the value as string (ex. `"true"`, not `true`), otherwise you
 
 ## service.beta.kubernetes.io/do-loadbalancer-http-idle-timeout-seconds
 
-Specifies the HTTP idle timeout configuration in seconds. If not specified, the default is 60 seconds.
\ No newline at end of file
+Specifies the HTTP idle timeout configuration in seconds. If not specified, the default is 60 seconds.
+
+## service.beta.kubernetes.io/do-loadbalancer-deny-rules
+
+Specifies the comma seperated DENY firewall rules for the load-balancer, supports IP and CIDR based rules
```

Comment:
We could make documentation slightly more maintainable by omitting this part about which types are exactly supported (and possibly reference some product documentation page instead) since we transparently pass the (comma-separated) annotation value to the LB API. That should allow us to automatically and always support the deny/allow rules that the backend supports without having to update the CCM.

Similar for the deny rule.